pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -190,6 +195,10 @@ func (c *ClusterConfig) ValidateVPCConfig() error {
 		if IsEnabled(c.VPC.AutoAllocateIPv6) {
 			return fmt.Errorf("auto allocate ipv6 is not supported with IPv6")
 		}
+	} else {
```

Comment:
This else hanging here after like 40-50 lines of code is super confusing and reads a bit uneasy.

Would you mind just doing a 

```go
if c.VPC.IPFamily == IPV4Family {
```

which reads a lot better and is more easy to follow along please? :)

Reply:
![image](https://user-images.githubusercontent.com/19669095/140767301-c71ba112-e01c-4148-8318-cbc53c8419e1.png)